common/config/rush/browser-approved-packages.json

```diff
@@ -2,6 +2,10 @@
 {
   "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/approved-packages.schema.json",
   "packages": [
+    {
+      "name": "normalize-path",
```

Comment:
When adding a new dependency, it will be patched into here. And if this dependency is not used in browser, it should be moved to `nonbrowser-approved-packages.json` by hand.

More info about this policy: https://rushjs.io/pages/maintainer/setup_policies/#approvedpackagespolicy-reviewing-new-npm-dependencies

Reply:
Got, will fix this json file manually